eframe/src/lib.rs

```diff
@@ -93,12 +101,13 @@ pub use web_sys;
 /// use wasm_bindgen::prelude::*;
 ///
 /// /// This is the entry-point for all the web-assembly.
-/// /// This is called once from the HTML.
+/// /// This is called from the HTML.
 /// /// It loads the app, installs some callbacks, then returns.
+/// /// It creates singleton app-handle that could be stopped calling `stop_web`
```

Comment:
```suggestion
/// /// It returns a handle to the running app that can be stopped calling `AppRunner::stop_web`.
```

I think it is also worth documenting whether or not it is safe to ignore the returned handle. As it is an `Arc`, I as a user would be afraid that dropping it would free the last handle, thus stopping the egui app.